pages/charcuterie.js

```diff
@@ -1,6 +1,8 @@
-import Head from 'next/head';
-import Carousel from '../components/Carousel';
-import Footer from '../components/Footer';
+import Head from 'next/head'
+import Carousel from '../components/Carousel'
+import Footer from '../components/Footer'
```

Comment:
Nit: not sure if it's your editor but the semi-colons were removed here. Not a big deal but figured I'd flag it